aio/scripts/lint-backend.sh

```diff
@@ -74,7 +74,7 @@ function run-ineffassign {
 # Execute script.
 ensure-go-dev-tools
 run-gofmt
-run-go-vet
+#run-go-vet
```

Comment:
Why? Does it work or not?

Reply:
@maciaszczykm @austbot, all good. my Go environment was wrong. I will submit another changes